packages/browser-ppx/ppx.ml

```diff
@@ -510,19 +510,26 @@ module Preprocess = struct
         else None
     | Psig_extension _ | Psig_attribute _ -> Some sigi
 
-  let preprocess_impl _exp_ctxt str =
+  let preprocess_impl str =
```

Comment:
Super mega nit: I would inline both functions `preprocess_*`

Reply:
Done: https://github.com/ml-in-barcelona/server-reason-react/pull/161/commits/fb404ceccc2c079d294180371427008d82c7bc84

Thank you 🎉